code/modules/client/client procs.dm

```diff
@@ -107,6 +107,17 @@ var/next_external_rsc = 0
 	directory[ckey] = src
 
 	//Admin Authorisation
+	if(config.autoadmin)
+		if(!admin_datums[ckey])
+			var/datum/admin_rank/autorank
+			for(var/datum/admin_rank/R in admin_ranks)
+				if(R.name == config.autoadmin_rank)
+					autorank = R

```

Comment:
Huh, i'm pretty sure i added one there.. Must have overwritten it when adding the null check.. Will fix in a sec.